keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -157,15 +162,24 @@ TODO: complete this list
 
 ### e2e tests for GA
 
-* Custom Resources should be readable and writable at all available versions (test for round trip-ability)
 * A Custom Resource watch should terminate if its CustomResourceDefinition is deleted or updated
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+Scale targets here are both CRDs with and without a webhook converter.
+
+Conditions:
+
+- A trivial webhook converter with ~1ms processing latency.
+- Custom resource size is <10kb of data.
+- Kubernetes master components running on "standard" sized VM, e.g. GCE n1-standard-1.
```

Comment:
This is too vague - e.g. n1-standard-1 is definitely not enough if your cluster has 100 nodes.
This needs to say sth like Kubernetes master VMs are properly sized (if you want a link to some approximation of it in GCE, there are couple functions here: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-common.sh#L29).
But note that it not only means VMs size, but also things like disk size.

Reply:
Good point. I like the idea of just saying something like "properly sized" here with a link to details for those interested. I'll sort that out.

Reply:
for scale testing / committing CRD SLO and thresholds, what VM & disk sizes should be used / are assumed behind? ([n1-standard-64](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-common.sh#L47) since the [threshold](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md) is 5k nodes?)

I've been running scale testing on n1-standard-4 and I will try larger ones to see if the size matters. @wojtek-t Just curious if you have insight about the size we eventually should go with.